### PR TITLE
CRUD country

### PR DESCRIPTION
**Pull request by SerjiKSM**

### CODE REVIEW

File: src/main/java/com/ksm/wstbackoffice/repository/CountryRepository.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.ksm.wstbackoffice.repository;
+
+import com.ksm.wstbackoffice.entity.Country;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface CountryRepository extends JpaRepository<Country, Long> {
```
Comment (Sabfir) [inline src/main/java/com/ksm/wstbackoffice/repository/CountryRepository.java]: Did you tested it? The Id is Long

**Review by Sabfir — CHANGES_REQUESTED** [review]

File: src/main/java/com/ksm/wstbackoffice/repository/CountryRepository.java (hunk 1 above)
Comment (SerjiKSM) [reply src/main/java/com/ksm/wstbackoffice/repository/CountryRepository.java]: done, fixed Long to String

File: src/main/java/com/ksm/wstbackoffice/service/CountryService.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+@Service
+public class CountryService {
+    private CountryRepository countryRepository;
+
+    public CountryService(CountryRepository countryRepository) {
+        this.countryRepository = countryRepository;
+    }
+
+    public List<Country> findAll() {
```
Comment (Sabfir) [inline src/main/java/com/ksm/wstbackoffice/service/CountryService.java:16]: Please also add method find by Id

Comment (SerjiKSM) [reply src/main/java/com/ksm/wstbackoffice/service/CountryService.java:16]: added

**Review by Sabfir — APPROVED** [review]

**Pull request merged**